src/modular/monty_form/lincomb.rs

```diff
@@ -39,7 +39,7 @@ mod tests {
         use rand_core::SeedableRng;
 
         let mut rng = rand_chacha::ChaCha8Rng::seed_from_u64(1);
-        for n in 0..1000 {
+        for n in 0..1500 {
```

Comment:
Probably would be more reliable to have a specialized regression test. And a proptest instead of iterating a 1000 times.